packages/config/src/config/build.js

```diff
@@ -54,7 +54,9 @@ export default () => ({
     },
     vueStyle: {}
   },
-  useForkTsChecker: process.env.NUXT_TS === 'true',
+  typescript: {
```

Comment:
Is this a breaking config change?

Reply:
yes, but I would say it's 100% TS related and as it's experimental, I don't know if it's important to make a depreciation around here

Reply:
@pi0

Reply:
Sure. We can just mention it in next release notes for proper upgrade guide :)

Reply:
Okay :)